src/components/CookiesBanner/index.tsx

```diff
@@ -245,6 +245,8 @@ const CookiesBanner = isDesktop
         cookiesAnalytics: boolean,
       ): void => {
         closeBanner()
+        cookiesSupportAndUpdates ? loadBeamer() : unloadBeamer()
+        cookiesAnalytics ? loadGoogleAnalytics() : unloadGoogleAnalytics()
```

Comment:
Do we need to check if GA/Beamer have been already loaded?
Otherwise it might include two scripts of each.

Reply:
Reverted this change to using `useEffect`